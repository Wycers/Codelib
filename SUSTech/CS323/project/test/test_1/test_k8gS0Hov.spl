int a, b, c;
float m, n;
char fuck;
struct TS
{
    int t;
    float s;
} testTs2;

int fibo(a)
{
    if (a == 1 || a == 2)
        return 1;
    return fibo(a - 1) + fibo(a - 2);
}

int main()
{
    int m, n, i;
    char hello;
    int ar[2][3];
    struct TS 1_testTs1;
    

    m = read();
    hello = 'a';
    hello2 = 'd';
    ar[0][1] = 1;
    i = 1.5f;
    testTs1.t = 0xz55ff;

    while (i <= m)
    {
        n = fibo(i);
        write(n);
        i++;
    }

    for (i = 1; i <= m; i = i + 1)
    {
        i = i + 1;
    }
    i += 1;

    return 1;
}