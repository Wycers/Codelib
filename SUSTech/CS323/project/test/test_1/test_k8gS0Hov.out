Error type B at Line 10: Missing specifier
Error type A at Line 22: unknown lexeme 1_testTs1
Error type B at Line 29: Missing semicolon ';'
Error type A at Line 30: unknown lexeme 0xz55ff
